packages/luigis-box/src/Model/Product/Parameter/ParameterRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\LuigisBoxBundle\Model\Product\Parameter;
+
+use Doctrine\ORM\EntityManagerInterface;
+use Shopsys\FrameworkBundle\Model\Product\Parameter\ParameterValue;
+
+class ParameterRepository
```

Comment:
at the moment, we have 4 classes named `ParameterRepository` in the project, I do not like that, to be honest.

I do not think you will like my suggestion but could we rename this class to something else, please? :smile: 
What about these?
- `LuigisBoxParameterRepository`
- `ParameterValueRepository`
- `ParameterValueProvider`
- `ParameterProvider`

Reply:
I have renamed it to ParameterValueRepository